charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -383,6 +383,13 @@ spec:
             # Enable eBPF dataplane mode.
             - name: FELIX_BPFENABLED
               value: "{{ .Values.config.felix.bpf.enabled }}"
+            # Move Kube-proxy healthz endpoint of calico-node to a different port when running in eBPF mode and kube-proxy is enabled.
+            {{- if and .Values.config.felix.bpf.enabled (not .Values.config.felix.bpfKubeProxyIPTablesCleanup.enabled) }}
+            - name: FELIX_BPFKUBEPROXYHEALTZPORT
```

Comment:
can we add a TODO here to remove this once the typo is fixed upstream? maybe add a link to the typo.

otherwise we lose context from the PR later and only see the two lines which may be confusing.

Reply:
I will add it in a follow-up PR